CONTRIBUTING.md

```diff
@@ -155,3 +159,18 @@ export FIRESTORE_EMULATOR_HOST="localhost:<Firebase emulator PORT>"
 - Every pull request should have associated issue(s) on our [issue tracker](https://github.com/Real-Dev-Squad/website-backend/issues).
 - For any non-trivial fixes and features, unit and integration tests must be added. The PR reviewer should not approve/merge PR(s) that lack these.
 - The PR(s) should be merged only after the CI passes.
+
+## Certain issues you may face while running the tests:
+
+- Firestore emulator is not starting
+- Java version is not above 11
+- When we run yarn test, it runs both the unit and integration tests (in this order). So after the unit tests are done, the java process is not killed automatically and when our integration test run it gives error.
+- Error: connect ECONNREFUSED ::1:8081
+
+## Possible solutions for above issues (in particular order):
+
+- add '--project=< your firestore app name>' flag to both the test scripts.
```

Comment:
Can you make it more explicit, like exactly where to add it.

```sh
firebase emulators:exec 'nyc mocha test/integration/tasks.test.js'  
firebase emulators:exec 'nyc --x=controllers --x=test --x=docs --x=mockdata mocha test/unit/**'
```

Reply:
Ok i ll do that.

Reply:
What I meant was something similar to this, my bad for badly convening.

```js

#!/bin/bash

# set 'test' environment
export NODE_ENV='test'

echo 'Start firestore emulator and run integration tests:'
firebase emulators:exec --project=<your firestore app name> 'nyc mocha test/integration/tasks.test.js'  


```

Reply:
Oh you want to me to add the placeholder in the script itself? @shmbajaj 
Ok i will do that.